### ISSUE TITLE
On the difference between Objects and Aliases

### ISSUE THREAD

**Issue by has2k1** [machow/quartodoc#317]
**Some Context**

Griffe provides the ability to distinguish between Objects and Aliases, where an Alias arises out of any reference to an object by a path that is not it's canonical_path. i.e. any imported Objects become Aliases.

Understandably, there after the difference between Objects and Aliases has to do with the names and paths.

**Issue**

In quartodoc, Objects and Aliases have another difference. By default, referring to an Object includes all members of the Object, while for an Alias the members of the target Object are excluded.

This means that in the config, using an Alias path is burdensome since we have to explicitly list the members. As an Alias path may be the recommended API path to an object, Aliases when listed in the config should be treated the same as Objects.

On the whole Aliases should only be treated differently if they are implicit.

For example.

1. In a config file, using an Alias is an explicit action
2. In a module, listing an Alias in `module.__all__` is an explicit action, since `__all__` defines the public members of the module.

**Demonstration of the problem**
```python
from quartodoc import blueprint, Auto,

doc1 = blueprint(Auto(name="pandas:DataFrame"))  # API Path (Alias)
doc2 = blueprint(Auto(name="pandas.core.frame:DataFrame"))  # Canonical Path (Object)

len(doc1.members), len(doc2.members)
```

```
0, 137
```

That means to document `pandas.DataFrame` and all of its members at the recommended API point, you would have to list 137 members.

**Comment by machow** [comment]
It seems like this is a bug, when filtering members of a Class (vs a Module).

Currently, for member selection:

* If neither of `include_imports` or `include_inherited` is true, then we remove any aliases
* But, we also ensure that methods of an alias are also aliases
* This causes us to exclude methods of a class alias

It sounds like even when the two flags above are false,  we still need to keep alias methods, so long as they point to methods on the class we're documenting. In contrast, they could be aliases to methods inherited from a parent--in which case the `include_inherited` flag should be respected.

## Small example

**some_class.py**

```python
class SomeClass:
    def x():
        "x func"
        ...

    def y():
        "y func"
        ...
```

**some_alias.py**

```python
from some_class import SomeClass
```

**script**

```python
from quartodoc import Auto, blueprint, MdRenderer, preview

# has methods as members
bp = blueprint(Auto(name = "some_alias.SomeClass", include_inherited=True))

# bug: doesn't have methods as members
bp = blueprint(Auto(name = "some_alias.SomeClass"))
```

**Comment by machow** [comment]
Here's a bit more beefed up version of **some_class.py**

```python
def external_func():
    "external"

class SomeParent:
    def zzz():
        "z func"

class SomeClass(SomeParent):
    def x():
        "x func"
        ...

    def y():
        "y func"
        ...

    external_func = external_func

```

**Comment by machow** [comment]
Should be addressed in v0.7.3
